.changelog/note.tmpl

```diff
@@ -0,0 +1,3 @@
+{{- define "note" -}}
+{{.Body}}{{if not (stringHasPrefix .Issue "_")}} [[GH-{{- .Issue -}}](https://github.com/hashicorp/terraform-provider-helm/issues/{{- .Issue -}})]{{end}}
```

Comment:
@BBBmau, thanks for making this change. I think there is a typo here and the actual URL should contain `terraform-provider-kubernetes`.